pkg/manager/http.go

```diff
@@ -974,7 +990,50 @@ func createPage(data any, body string) *template.Template {
 	{{HEAD}}
 </head>
 <body>
-%v
+	<header id="topbar">
+		<table class="position_table">
+			<tr>
+				<td>
+					<h1><a href="/">syzkaller</a></h1>
+				</td>
+				<td>
+					<form action="/action" method="post">
+						<input type="hidden" name="url" value="{{.CurrentURL}}" />
+						<button type="submit" name="action" value="toggle-expert" class="action_button{{if .ExpertMode}}_selected{{end}}" title="Toggle expert mode">
+							🧠
+						</input>
+					</form>
+				</td>
+				<td class="search">
+					<a href="https://github.com/google/syzkaller/blob/master/docs/" target="_blank">docs</a> |
+					<a href="https://groups.google.com/forum/#!forum/syzkaller" target="_blank">mailing list</a> |
+					<a href="{{.GitRevisionLink}}" target="_blank">source {{.GitRevision}}</a>
+				</td>
+			</tr>
+		</table>
+		<div class="navigation">
+			<div class="navigation_tab{{if eq .URLPath "/stats"}}_selected{{end}}">
+				<a href='/stats'>📈 stats</a>
+			</div>
+			<div class="navigation_tab{{if eq .URLPath "/cover"}}_selected{{end}}">
+				<a href='/cover'>📃 coverage</a>
+			</div>
+			<div class="navigation_tab{{if eq .URLPath "/syscalls"}}_selected{{end}}">
+				<a href='/syscalls'>🤖 syscalls</a>
+			</div>
+			<div class="navigation_tab{{if eq .URLPath "/corpus"}}_selected{{end}}">
+				<a href='/corpus'>🛒 corpus</a>
+			</div>
+			<div class="navigation_tab{{if eq .URLPath "/vms"}}_selected{{end}}">
+				<a href='/vms'>💻 VMs</a>
+			</div>
+			<div class="navigation_tab{{if eq .URLPath "/config"}}_selected{{end}}">
+				<a href='/config'>🔧 config</a>
+			</div>
+		</div>
+	</header>
+	<br>
+	%v
```

Comment:
This template is too big now. Let's move it to the .html file. It'll additionally enable the formatting checks you've added yesterday.

Reply:
Done.